test/truefi/LoanToken.test.ts

```diff
@@ -204,6 +205,16 @@ describe('LoanToken', () => {
       expect(await loanToken.status()).to.equal(LoanTokenStatus.Settled)
     })
 
+    it('loan can be payed back 1 day after term has ended', async () => {
+      await loanToken.fund()
+      await withdraw(borrower)
+      await timeTravel(provider, yearInSeconds + dayInSeconds / 2)
+      await expect(loanToken.close()).to.be.revertedWith('LoanToken: Borrower can still pay the loan back')
+      await tusd.mint(loanToken.address, parseEth(1100))
```

Comment:
Maybe let's use explicit call `repay` call here instead of just pouring tokens into the contract?